### PR TITLE
{lang}[foss/2018a,intel/2018a] Python v2.7.14

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)
requires ~~#5676~~

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2084.delcatty.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/9d091e32f97df74b5d76c751fa2349eb for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2438.golett.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/cdb60ceacf6b3747f3b8244ff08623c1 for a full test report.

File: easybuild/easyconfigs/p/Python/Python-2.7.14-foss-2018a.eb [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+    ('numpy', '1.14.0', {
+        'patches': ['numpy-1.12.0-mkl.patch'],
+        'source_tmpl': '%(name)s-%(version)s.zip',
+        'source_urls': ['https://pypi.python.org/packages/source/n/numpy/'],
+        'checksums': [
+            '3de643935b212307b420248018323a44ec51987a336d1d747c1322afc3c099fb',  # numpy-1.14.0.zip
+            'f212296ed289eb1b4e3f703997499dee8a2cdd0af78b55e017477487b6377ca4',  # numpy-1.12.0-mkl.patch
+        ],
+    }),
+    ('scipy', '0.19.1', {
```
Comment (migueldiascosta) [inline easybuild/easyconfigs/p/Python/Python-2.7.14-foss-2018a.eb]: why not `1.0.0`, as in the `2018.01` toolchains?

**Comment by migueldiascosta** [comment]
Test report by @migueldiascosta
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
grc-cluster1 - Linux centos 6.9, Intel(R) Xeon(R) CPU E5-2640 0 @ 2.50GHz, Python 2.6.6
See https://gist.github.com/2d2acea9e4bc982a83e38aedd0286e4e for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2500.golett.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/1ff55a0209bb8e85f5faf9116e153e97 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2104.delcatty.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/dbb4d0762b2bce5670c525c1c6168eaf for a full test report.

**Comment by verdurin** [comment]
Test report by @verdurin
**FAILED**
Build succeeded for 9 out of 17 (2 easyconfigs in this PR)
dell.lan - Linux fedora 27, Intel(R) Core(TM) i7-4600U CPU @ 2.10GHz, Python 2.7.14
See https://gist.github.com/24d638376b22d2edbe63a99dd3bf66a0 for a full test report.

**Comment by vanzod** [comment]
Test report by @vanzod
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
cermis - Linux debian 9.3, Intel(R) Xeon(R) CPU           E5620  @ 2.40GHz, Python 2.7.13
See https://gist.github.com/8058e4c8ec5abb19a66e8cc5809d44ce for a full test report.

**Comment by vanzod** [comment]
Going in, thanks @boegel!

**Pull request merged**